### PR TITLE
[#7] Fix env and add relude

### PR DESCRIPTION
**Pull request by willbasky**
Resolves [#7](https://github.com/kowainik/issue-wanted/issues/7)

### CODE REVIEW

**Comment by willbasky** [comment]
```
notifications:
  email: false
  slack:
    rooms:
      - kowainik: ???
    on_success: change
    on_failure: always
```
It needs new key for travis.

File: src/Prelude.hs [hunk 1]
```diff
@@ -1,8 +0,0 @@
--- | Uses [universum](https://hackage.haskell.org/package/universum) as default Prelude.
-
-module Prelude
-       ( module Universum
```
Comment (vrom911) [inline src/Prelude.hs]: I guess it's fine to use `base-noprelude` and keep this module replacing `Universum` to `Relude` 🙂 
It allows not to add `import Relude` line to each file.

File: issue-wanted.cabal [hunk 2]
```diff
@@ -1,5 +1,6 @@
+cabal-version:       2.2
 name:                issue-wanted
-version:             0.0.0
```
Comment (vrom911) [inline issue-wanted.cabal]: We can still keep 0.0.0 as the start point

File: .travis.yml [hunk 3]
```diff
@@ -1,48 +1,83 @@
...
+    else
+      stack build --test --no-terminal
+    fi
 
 notifications:
   email: false
   slack:
     rooms:
-      - kowainik:k3hrOZtFA7ViWPFCLrSokLcU
+      - kowainik:n60CMbB0kfJHRmYKIPmjf7Tp
```
Comment (vrom911) [inline .travis.yml]: We can keep the previous room number (it's #issue-wanted-apps in our Slack, at least it should be so)

File: issue-wanted.cabal [hunk 4]
```diff
@@ -28,7 +28,7 @@ library
                        -Wredundant-constraints
                        -fhide-source-paths
 
-  build-depends:       base >= 4.10 && < 5
+  build-depends:       base-noprelude
```
Comment (vrom911) [inline issue-wanted.cabal]: You should add >= 4.11 && < 5

File: .travis.yml [hunk 5]
```diff
@@ -10,19 +10,6 @@ cache:
   - "$HOME/.stack"
   - "$TRAVIS_BUILD_DIR/.stack-work"
 
-matrix:
-  include:
```
Comment (vrom911) [inline .travis.yml]: Oops! I guess it was deleted by mistake

File: stack.yaml [hunk 6]
```diff
@@ -1,8 +1,7 @@
-resolver: lts-11.10
+resolver: lts-12.09
 
 extra-deps:
-  - base-noprelude-4.10.1.0
-  - universum-1.2.0
+  - relude-0.3.0
 
 ghc-options:
   "$locals": -fhide-source-paths
```
[the remark this answers is not in the document]
Comment (willbasky) [reply stack.yaml]: Then error occurs 
```
In the dependencies forissue-wanted-0.0.0:
base-noprelude must match>=4.11.1.0 && <5, but the stack configuration has
                   no specified version  (latest matching version is4.11.1.0)
relude-0.1.1 from stack configuration does not match>=0.3.0 && <0.4
                 (latest matching version is0.3.0)
needed sinceissue-wanted is a build target.
```

[the remark this answers is not in the document]
Comment (vrom911) [reply stack.yaml]: I guess he meant this line:
```
ghc-options:	ghc-options:
  "$locals": -fhide-source-paths
```

File: issue-wanted.cabal [hunk 7]
```diff
@@ -28,8 +28,9 @@ library
                        -Wredundant-constraints
                        -fhide-source-paths
 
-  build-depends:       base-noprelude
+  build-depends:       base-noprelude >= 4.11 && < 5
                      , relude ^>= 0.3.0
+                     , base-noprelude >= 4.11.1.0
```
Comment (vrom911) [inline issue-wanted.cabal]: duplicate 🙂

File: issue-wanted.cabal [hunk 8]
```diff
@@ -13,16 +14,23 @@ category:            Web, Application
...
                    , CHANGELOG.md
-cabal-version:       1.24
-tested-with:         GHC == 8.2.2
+tested-with:         GHC == 8.4.3
 
 library
   hs-source-dirs:      src
   exposed-modules:     IssueWanted
-                       Prelude
+                     , Prelude
```
Comment (ChShersh) [inline issue-wanted.cabal]: You don't need commas in `exposed-modules`

**Pull request merged**